src/components/page-scheduled-job/index.js

```diff
@@ -80,6 +80,13 @@ const PageScheduledJob = (props) => {
                     </strong>
                   </Typography>
                 )}
+              {scheduledJob?.message && (
+                <>
+                  <Typography>
+                    <strong>{scheduledJob.message}</strong>
+                  </Typography>
+                </>
+              )}
```

Comment:
Display status text in a \<Code\> component and remove unnecessary Fragment blocks
```suggestion
              {scheduledJob?.message && (
                <Code>
                  <strong>{scheduledJob.message}</strong>
                </Code>
              )}
```
![suggest_z4eg](https://user-images.githubusercontent.com/32248439/141278982-4f9b30fe-28a1-4d88-966b-4e2d8f0fcd76.png)